### DISCUSSION TITLE
After updating to next 12.3.4 from next 12.0.4 everything's fine in local but my deploy links crashes browsers

### DISCUSSION THREAD

**Discussion by kutaybuyukkorukcu** [vercel/next.js#44525] [Help]
Hey all,

I've been trying to solve this one problem. I also don't get any errors from the app, only crashes like page unresponsive (chrome)

I didn't have any problem like this one after upgrading from v12 to v12.1, will try upgrading to v12.2 now to see if it's related with v12.3

**Comment by icyJoseph** [comment]
Sound like, perhaps, some minification of your code done by SWC is causing an infinite loop, which is super hard to debug, but I have seen it before. 

Do you see the same issue if you `next build` and `next start`? 

 ## Edit

OP was swapping React with Preact, which has been observed to cause issues.

**Reply by kutaybuyukkorukcu** [reply]
Yep, I see the same issue with next build and next start afterwards. I've also set swcMinify as false

**Reply by icyJoseph** [reply]
Alright that's good. Are you doing any other "out of the normal" things? Like swapping React with Preact, etc? Locally, can you try and remove pages until the program becomes responsive again? Or try to remove features from `_app`.

**Reply by kutaybuyukkorukcu** [reply]
Silly me I forgot to reply, It was because of React swapping with Preact 🙌
